snakefile.py

```diff
@@ -136,13 +143,15 @@ def tool(name):
         r1 = os.path.join(READS_DIR, "{sample}_R1.fastq"),
         r2 = os.path.join(READS_DIR, "{sample}_R2.fastq")
     output:
-        r1 = os.path.join(TRIMMED_READS_DIR, "{sample}_1.fastq"),
-        r2 = os.path.join(TRIMMED_READS_DIR, "{sample}_2.fastq")
+        r1 = os.path.join(TRIMMED_READS_DIR, "{sample}_trimmed_R1.fastq"),
+        r2 = os.path.join(TRIMMED_READS_DIR, "{sample}_trimmed_R2.fastq")
     params:
         len_cutoff = int(150 * 0.8), # read length * pct_cutoff
-        output = os.path.join(TRIMMED_READS_DIR, "{sample}")
+        output = os.path.join(TRIMMED_READS_DIR, "{sample}"),
+        tmp_r1 = os.path.join(TRIMMED_READS_DIR, "{sample}_1.fastq"),
+        tmp_r2 = os.path.join(TRIMMED_READS_DIR, "{sample}_2.fastq")
     log: os.path.join(LOG_DIR, 'prinseq_{sample}.log')
-    shell: "{PRINSEQ_EXEC} -fastq {input.r1} -fastq2 {input.r2} -ns_max_n 4 -min_qual_mean 30 -trim_qual_left 30 -trim_qual_right 30 -trim_qual_window 10 -out_good {params.output} -out_bad null -min_len {params.len_cutoff} >> {log} 2>&1"
+    shell: "{PRINSEQ_EXEC} -fastq {input.r1} -fastq2 {input.r2} -ns_max_n 4 -min_qual_mean 30 -trim_qual_left 30 -trim_qual_right 30 -trim_qual_window 10 -out_good {params.output} -out_bad null -min_len {params.len_cutoff} >> {log} 2>&1 && sleep 10 && mv {params.tmp_r1} {output.r1} && mv {params.tmp_r2} {output.r2}"
```

Comment:
I know that other pipelines have this `sleep 10`, but I'd suggest to leave it off.  There's no good reason to add it.

Reply:
Ok, I'll remove it.